### ISSUE TITLE
List.Item requiring `tvParallaxProperties` and `hasTVPreferredFocus`

### ISSUE THREAD

**Issue by bombillazo** [callstack/react-native-paper#3002]
### Current behaviour

Any `List.Item` component created is required `tvParallaxProperties` and `hasTVPreferredFocus` to be set

### Expected behaviour

I can use `List.Item` without setting those props

### Code sample

```jsx
<List.Item title="Name" />
```

### Screenshots (if applicable)

![image](https://user-images.githubusercontent.com/44002321/145637788-b4147faf-2f95-460d-9061-9adcedc228bd.png)

### What have you tried
Currently I am setting dummy/default values.
```jsx
<List.Item
  tvParallaxProperties={{}}
  hasTVPreferredFocus={false}
  title="Name"
/>
```

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        |
| react-native          | 0.66.2
| react-native-paper    |  4.10.0
| node                  | 14.17.3
| npm or yarn           |
| expo sdk              |

**Comment by lukewalczak** [comment]
Hello @bombillazo, could you please try to use version 4.10.1 or the latest one with rn types 0.66.x?

**Comment by bombillazo** [comment]
Ok seems this is fixed in the latest version: 4.11.0
